README.md

```diff
@@ -20,11 +20,11 @@ To convert the DICOM dataset in a BIDS structure for this project, run the follo
 ~~~
 curate_project.py -path-in <path_DICOM_dataset> -path-output <path_BIDS_dataset>
 ~~~
-The BIDS datset is (for now) in `duke: temp/uk_biobank_BIDS`
+The BIDS datset is (for now) in **|TODO : update with raw and processed repo**
```

Comment:
ah! maybe that TODO addresses my comment below https://github.com/sct-pipeline/ukbiobank-spinalcord-csa/pull/30/files#r565672217

Reply:
haha yes! I am waiting for when we officially have  both datasets to update the info about them!